docs/book/src/topics/dual-stack.md

```diff
@@ -0,0 +1,69 @@
+# Dual-stack clusters
+
+## Overview
+
+CAPZ enables you to create [dual-stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) Kubernetes cluster on Microsoft Azure.
+
+- Dual-stack support is available for Kubernetes version 1.21.0 and later on Azure.
+
+To deploy a cluster using dual-stack, use the [dual-stack flavor template](../../../../templates/cluster-template-dual-stack.yaml).
+
+Things to try out after the cluster created:
+
+- Nodes have 2 internal IPs, one from each IP family.
+
+```bash
+kubectl --kubeconfig=/tmp/capi-dual-stack.kubeconfig get node capi-dual-stack-md-0-j96nr -o go-template --template='{{range .status.addresses}}{{printf "%s: %s \n" .type .address}}{{end}}'
+Hostname: capi-dual-stack-md-0-j96nr 
+InternalIP: 10.1.0.4 
+InternalIP: 2001:1234:5678:9abd::4 
+```
+
+- Nodes have 2 `PodCIDRs`, one from each IP family.
+
+```bash
+kubectl get node capi-dual-stack-md-0-j96nr -o go-template --template='{{range .spec.podCIDRs}}{{printf "%s\n" .}}{{end}}'
+10.244.2.0/24
+2001:1234:5678:9a42::/64
+```
+
+- Pods have 2 `PodIP`, one from each IP family.
+
+```bash
+kubectl get pods nginx-7f76bfdf58-ftmb6 -o go-template --template='{{range .status.podIPs}}{{printf "%s \n" .ip}}{{end}}' 
```

Comment:
Small thing: since these commands can be copy/pasted for users to try, we can change the name of a pod to something like `<pod-name>` to fit w/ doc styling.

Reply:
done!